apm-agent-plugins/apm-servlet-plugin/src/test/java/co/elastic/apm/agent/servlet/ApmFilterTest.java

```diff
@@ -235,7 +235,7 @@ void captureTransactionNameBasedOnServlet() throws IOException, ServletException
         filterChain = new MockFilterChain(new TestServlet());
         filterChain.doFilter(new MockHttpServletRequest("GET", "/foo"), new MockHttpServletResponse());
         assertThat(reporter.getTransactions()).hasSize(1);
-        assertThat(reporter.getFirstTransaction().getName().toString()).isEqualTo("ApmFilterTest$TestServlet#doGet");
+        assertThat(reporter.getFirstTransaction().getName().toString()).isEqualTo("GET ApmFilterTest$TestServlet");
```

Comment:
I'm not sure if I like this format better. For other transaction names, we use either `$method $path`, like `GET /test` or `$class#$method`, like `MyServlet#doGet`. This is some sort of mix, like `$method $class`.